include/nuttx/sched.h

```diff
@@ -532,6 +526,12 @@ struct tcb_s
 
   FAR struct task_group_s *group;      /* Pointer to shared task group data */
 
+  /* Address Environment ****************************************************/
+
+#ifdef CONFIG_ARCH_ADDRENV
+  FAR struct addrenv_s *mm_own;         /* Task (group) own memory mappings */
```

Comment:
mm_own->addrenv

Reply:
Should I change mm_curr to addrenv_curr too ?

Reply:
could be, I think addrenv is better prefix than mm since many place use addrenv.

Reply:
Done